src/rewritable_units/element.rs

```diff
@@ -489,7 +489,7 @@ impl<'r, 't> Element<'r, 't> {
 
     /// Sets a handler to run when the end tag is reached.
     ///
-    /// Subsequent calls to the method on the same element replace the previous handler.
+    /// Multiple handlers can be registered and will run in the order of registration.
```

Comment:
Need to clarify here that consecutive calls to the function add more handlers

Reply:
:heavy_check_mark: